### PR TITLE
Apex Pills

### PR DESCRIPTION
**Pull request by lxm6938amount**
Updating neutral pill component

### CODE REVIEW

**Comment by estenp** [comment]
@JohnathanHuggett @pragati-amount Are we ok with this variant names in terms of how they will be used as variant prop to the Pill component?
e.g.
`variant="info-outlined"`

File: tokens/apex/components/unrestricted/pill/counterAttention.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (estenp) [inline tokens/apex/components/unrestricted/pill/counterAttention.json:1]: @pragati-amount @JohnathanHuggett  I don't think these "counter" variants are able to be achieved within the Pill component's constraints, and I'm not sure exactly what they are.

I had to fix Liam's initial JSON to use Pill token constraints, so they are likely not achieving design with these tokens.

We need to either ensure they are styled using valid Pill tokens in Figma or remove these as Pill variants and audit this as it's own component.

**Review by estenp — APPROVED** [review]

File: tokens/apex/components/unrestricted/pill/counterAttention.json (hunk 1 above)
Comment (JohnathanHuggett) [reply tokens/apex/components/unrestricted/pill/counterAttention.json:1]: All these should be removed. They are being explored. I am having @pragati-amount remove them from the pills file

**Comment by JohnathanHuggett** [comment]
> @JohnathanHuggett @pragati-amount Are we ok with this variant names in terms of how they will be used as variant prop to the Pill component? e.g. `variant="info-outlined"`

Yes that seems good

**Review by JohnathanHuggett — APPROVED** [review]

**Review by JohnathanHuggett — CHANGES_REQUESTED** [review]
need to remove the counter variants

**Review by JohnathanHuggett — APPROVED** [review]

**Pull request merged**